src/script/cpp_api/s_base.cpp

```diff
@@ -35,6 +35,7 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 
 extern "C" {
 #include "lualib.h"
+#include "bit.c"
```

Comment:
uhh

Reply:
Changed it to not use it when luajit is used because of "double build"

Reply:
That wasn't my complaint here, `.c` files are not supposed to be included somewhere else but rather added to CMakeLists.txt like all other files.

Reply:
So I need to create an extra .h file for it?

Reply:
You would, yes

Reply:
Done